src/main/java/net/minestom/server/listener/BlockPlacementListener.java

```diff
@@ -131,6 +131,7 @@ public static void listener(ClientPlayerBlockPlacementPacket packet, Player play
         // BlockPlacementRule check
         Block resultBlock = playerBlockPlaceEvent.getBlock();
         final BlockPlacementRule blockPlacementRule = BLOCK_MANAGER.getBlockPlacementRule(resultBlock);
+        if(placementPosition.y() >= 320) return;
```

Comment:
You should probably not hardcode the number, use the dimension to get the maximum y value. Also you should probably move this to just after the initialization of `placementPosition` for a fast exit